models/DataObject/ClassDefinition/Layout/DynamicTextLabelInterface.php

```diff
@@ -23,7 +23,7 @@ interface DynamicTextLabelInterface
 {
     /**
      * @param string $data as provided in the class definition
-     * @param Concrete $object
+     * @param Concrete|null $object
```

Comment:
This can't be null.
The object param in Pimcore\Model\DataObject\ClassDefinition\Layout\Text::enrichLayoutDefinition can be null now.

Reply:
In the current implementation it can't be null, true. 

Better to move the change in the documentation into a separate task for 7.0? IMHO, it should be able to handle null values as well solely based on the $params. Purpose could be a "preview" or whatever.